### ISSUE TITLE
Configuration files directory differs from Vanilla Doom

### ISSUE THREAD

**Issue by Henrique194** [chocolate-doom/chocolate-doom#1655]
### Background

Version of Chocolate Doom: 3.0.1

Operating System and version: Windows CE or any OS different than Windows

Game: (Doom/Heretic/Hexen/Strife/other): All

Any loaded WADs and mods (please include full command line): No WADs or mods

### Bug description

Observed behavior:
Under Windows CE or any OS different than Windows, configuration files are saved in a directory specified by SDL, instead of saving them in the same directory as the executables.

Expected behavior:
Behave as Vanilla Doom for any OS, saving configuration files in the same directory as the executables.

**Comment by fabiangreffrath** [comment]
> Expected behavior: Behave as Vanilla Doom for any OS, saving configuration files in the same directory as the executables.

That'd be `/usr/bin` in Linux or any other unixoid OS and that directory is generally not writable for everyone, for good reasons.

**Comment by Henrique194** [comment]
@fabiangreffrath Good point. Maybe we can allow Windows CE to behave as Vanilla then? It would only require us to remove the `defined(_WIN32_WCE)` from this [line](https://github.com/chocolate-doom/chocolate-doom/blob/22510a8be1b45a08f9d1ad341bd8f2d499502051/src/m_config.c#L2530).

**Comment by fabiangreffrath** [comment]
Is this really about Windows CE or is it about the principle?

**Comment by Henrique194** [comment]
It is more about simplifying the code and bringing Window CE behavior closer to the original Doom.
